repos.yml

```
@@ -5336,6 +5336,10 @@ repos:
     group: deepin-sysdev-team
     info: Nvidia driver
 
+  - repo: open-gpu-kernel-modules
```

```suggestion
  - repo: nvidia-open-gpu-kernel-modules
```

https://tracker.debian.org/pkg/nvidia-open-gpu-kernel-modules